samcli/lib/build/app_builder.py

```diff
@@ -641,7 +641,7 @@ def _build_function(  # pylint: disable=R1710
                     f"update to a newer supported runtime. For more information please check AWS Lambda Runtime "
                     f"Support Policy: https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html"
                 )
-                LOG.warning(self._colored.yellow(message))
+                LOG.warning(self._colored.color_log(msg=message, color="yellow"), extra=dict(markup=True))
```

Comment:
Can we make this inside the colored instance? So that we don’t need to have string param as yellow and can use existing methods for colorful printing?

Reply:
I would rather keep things in the utility methods (or in other ways) where we don't need to re-write same parameters (like `extra` param) for logging in different colors.

Maybe we should use `click` methods to write colorful output, and logging should be pure text.